### PR TITLE
RHEL-72207: Apply clang-format to ivshmem folder

### PR DESCRIPTION
**Pull request by Jedoku**

### CODE REVIEW

**Review by YanVugenfirer — APPROVED** [review]

**Comment by YanVugenfirer** [comment]
@Jedoku The PR has conflicts. Otherwise +1

**Review by YanVugenfirer — APPROVED** [review]

File: ivshmem/Device.c [hunk 1]
```diff
@@ -167,55 +161,71 @@ NTSTATUS IVSHMEMEvtDevicePrepareHardware(_In_ WDFDEVICE Device, _In_ WDFCMRESLIS
...
+                               memIndex,
+                               descriptor->u.Memory.Start.QuadPart,
+                               deviceContext->shmemAddr.PhysicalAddress.QuadPart,
+                               descriptor->u.Memory.Length,
+                               deviceContext->shmemAddr.NumberOfBytes);
 
 #if (NTDDI_VERSION >= NTDDI_WIN8)
-                result = MmAllocateMdlForIoSpace(&deviceContext->shmemAddr, 1, &deviceContext->shmemMDL);
+                    result = MmAllocateMdlForIoSpace(&deviceContext->shmemAddr, 1, &deviceContext->shmemMDL);
 #else
```
Comment (kostyanf14) [inline ivshmem/Device.c:178]: @YanVugenfirer 

Looks like a part for refactoring. This condition can be removed as we drop Win8 support.

File: ivshmem/Device.c [hunk 2]
```diff
@@ -225,21 +235,24 @@ NTSTATUS IVSHMEMEvtDevicePrepareHardware(_In_ WDFDEVICE Device, _In_ WDFCMRESLIS
             }
 
             if (++deviceContext->interruptsUsed == 65)
-              DEBUG_INFO("%s", "This driver does not support > 64 interrupts, they will be ignored in the ISR.");
+                DEBUG_INFO("%s", "This driver does not support > 64 interrupts, they will be ignored in the ISR.");
```
Comment (kostyanf14) [inline ivshmem/Device.c:238]: Why without brackets?

File: ivshmem/Queue.c [hunk 3]
```diff
@@ -209,167 +196,167 @@ VOID IVSHMEMEvtDeviceFileCleanup(_In_ WDFFILEOBJECT FileObject)
...
+                                   const size_t OutputBufferLength,
+                                   const WDFREQUEST Request,
+                                   size_t *BytesReturned,
+                                   BOOLEAN ForKernel)
 {
     WDFFILEOBJECT fileObject = WdfRequestGetFileObject(Request);
     PFILE_CONTEXT fileContext = FileGetContext(fileObject);
     // only one mapping per driver handle is allowed
     if (fileContext->shmemMap)
         return STATUS_DEVICE_ALREADY_ATTACHED;
```
Comment (kostyanf14) [inline ivshmem/Queue.c:260]: brackets?

File: ivshmem/test/ivshmem-test.cpp [hunk 4]
```diff
@@ -3,251 +3,314 @@
...
+
+        break;
+    }
+
+    if (devHandle != INVALID_HANDLE_VALUE)
+        CloseHandle(devHandle);
+
+    if (infData)
+        free(infData);
+
```
Comment (kostyanf14) [inline ivshmem/test/ivshmem-test.cpp:312]: brackets?

**Review by kostyanf14 — CHANGES_REQUESTED** [review]

File: ivshmem/Device.c (hunk 1 above)
Comment (kostyanf14) [reply ivshmem/Device.c:178]: @Jedoku ignore this comment

**Comment by kostyanf14** [comment]
s/clang-formatt/clang-format/

**Review by YanVugenfirer — APPROVED** [review]

**Review by YanVugenfirer — APPROVED** [review]

**Review by YanVugenfirer — APPROVED** [review]

**Review by kostyanf14 — APPROVED** [review]

**Pull request merged**